src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -22839,15 +22839,15 @@ public MyAttr(int p) {}
             );
     }
 
-    [Fact(Skip = "Cycle")] // PROTOTYPE: There is a cycle due to the attribute
+    [Fact]
     public void ReceiverParameterScope_07_InAttribute()
     {
         var src = @"
 static class Extensions
 {
     extension(int p)
     {
-        [System.Runtime.CompilerServices.IndexerName(nameof(p))]
+        [MyAttr(nameof(p))]
```

Comment:
Could we keep the original test with `[IndexerName(nameof(p))]`?

Reply:
> Could we keep the original test with `[IndexerName(nameof(p))]`?

The purpose of the test is to confirm that use of receiver parameter in an attribute inside `nameof` is allowed. Keeping the original code defeats this purpose because of the limitations applied to `IndexerName` attribute specifically. That limitation is tested by other dedicated test.